extensions/file/src/test/java/org/mule/extension/file/FileConnectorTestCase.java

```diff
@@ -63,33 +68,42 @@ protected void assertExists(boolean exists, File... files) {
   }
 
   protected Event readHelloWorld() throws Exception {
-    return getPath(HELLO_PATH);
+    return getPath(HELLO_PATH).run();
   }
 
-  protected Message readPath(String path) throws Exception {
-    return getPath(path).getMessage();
+  protected FlowRunner readPath(String path) throws Exception {
+    return getPath(path);
   }
 
-  protected Event getPath(String path) throws Exception {
-    return flowRunner("read").withVariable("path", path).run();
+  protected FlowRunner getPath(String path) throws Exception {
+    return flowRunner("read").withVariable("path", path);
   }
 
   protected String readPathAsString(String path) throws Exception {
-    return IOUtils.toString((AbstractFileInputStream) readPath(path).getPayload().getValue());
+    return IOUtils.toString((AbstractFileInputStream) readPath(path).run().getMessage().getPayload().getValue());
   }
 
-  protected void doWrite(String path, Object content, FileWriteMode mode, boolean createParent) throws Exception {
-    doWrite("write", path, content, mode, createParent);
+  protected FlowRunner doWrite(String path, Object content, FileWriteMode mode, boolean createParent) throws Exception {
+    return doWrite("write", path, content, mode, createParent);
```

Comment:
this method is called doWrite and yet after invoking it you haven't wrote anything. What is this refactor about? it doens't seem to make any sense